target/linux/rockchip/image/mmc.bootscript

```diff
@@ -0,0 +1,8 @@
+part uuid mmc ${devnum}:2 uuid
+
+setenv bootargs "console=ttyS2,1500000 console=tty1 earlycon=uart8250,mmio32,0xff1a0000 root=PARTUUID=${uuid} rw rootwait"
```

Comment:
can we make 0xff1a0000 from earlycon as a variable and pass it by Makefile ? for rk3328 we need 0xff130000 so we can have only one mmc script

Reply:
Yes, but please do that in your own PR.